src/modm/platform/dma/stm32/dma_hal.hpp.in

```diff
@@ -22,6 +23,13 @@ namespace modm
 namespace platform
 {
 
+
+#ifndef DMA2_BASE
+# define DMA2_BASE 0
+# define DMA2_Channel1_BASE 0
+# define DMA2_CSELR_BASE
+#endif
+
```

Comment:
```
#ifndef DMA2_BASE
 # define DMA2_BASE 0
```
So the check in `dma.hpp` is always true?

Reply:
Yeah 🤦🏽 

I replace the macros with jinja magic anyways...